zanata-war/src/main/java/org/zanata/action/ProjectHome.java

```diff
@@ -1063,8 +1063,8 @@ private void ensureMutualExclusivity(
     /**
      * Update the about page to the entered value, and show a success message.
      */
-    @Restrict("#{s:hasPermission(projectHome.instance, 'update')}")
     public void updateAboutPage() {
+        identity.checkPermission(instance, "update");
```

Comment:
In fact, we could just override the super.update() method in EntityHome. What do you think?